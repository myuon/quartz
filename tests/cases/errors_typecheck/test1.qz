fun t() {
    let s = 10;
    println(s);
}

