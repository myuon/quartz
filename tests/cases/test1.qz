fun main(): i32 {
    return 1;
}

