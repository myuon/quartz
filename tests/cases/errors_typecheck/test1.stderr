TypeError unification failed: i32 != string at main[L3:12-L3:13]

2 |     let s = 10;
3 |     println(s);
3 |             ^
4 | }
